repo/packages/C/chronos/0/marathon.json

```diff
@@ -28,6 +28,8 @@
       "portIndex": 0,
       "protocol": "HTTP"
     }
-  ]
-
+  ],
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{chronos/mesos-framework-name}}"
```

Comment:
@ConnorDoyle, I was using Cassandra as an example. It seems to be the current workaround we are using? https://github.com/mesosphere/universe/blob/master/repo/packages/C/cassandra/0/marathon.json#:L34:L36 cc @BenWhitehead